lib/kernel.js

```diff
@@ -101,7 +101,10 @@ module.exports = function(config, cb) {
 
   function stop() {
```

Comment:
should close be called on these services? https://github.com/nearform/nscale-kernel/blob/master/lib/containers.js#L51